src/utils.ts

```diff
@@ -321,6 +320,23 @@ export async function expectError(script: Promise<any>) {
   expect(isError).toBe(true)
 }
 
+export async function expectErrorCode(errorCode: bigint, script: Promise<any>) {
```

Comment:
It should be definetly in `testUtils.ts`. You can enhance type of errorCode with `InvariantError | FeeTiersError`. We don't need to keep two utils to assert errors. One with interface
`expectError(script: Promise<any>, errorCode?: ...) { ... }` should be enough

Reply:
I will handle that in a separate feature branch basing off master